### PR TITLE
Fix Editor examples (#358)

### PR DESCRIPTION
**Pull request by jaydenmilne**
We hadn't updated them for the scenario config changes.

### CODE REVIEW

File: example.py [hunk 1]
```diff
@@ -169,7 +221,3 @@ def editor_multi_agent_example():
         for _ in range(1000):
             states = env.tick()
 
-
-if __name__ == "__main__":
```
Comment (kolbytn) [inline example.py]: I'm not against removing this, but I just want to make sure it was intentional.

File: example.py [hunk 2]
```diff
@@ -135,12 +135,41 @@ def world_command_examples():
...
     """
-    agent_sensors = [sensors.RGBCamera, sensors.LocationSensor, sensors.VelocitySensor]
-    agent = AgentDefinition("uav0", agents.UavAgent, agent_sensors)
-    env = HolodeckEnvironment([agent], start_world=False)
-    env.agents["uav0"].set_control_scheme(1)
+
+    config = {
+        "name": "test",
+        "world": "TestWorld",
+        "main_agent": "uav0",
```
Comment (kolbytn) [inline example.py]: We updated the config files to require package_name. Has that change been merged yet and should be reflected here? Just checking.

Comment (MaxRobinsonTheGreat) [reply example.py]: It does not need to be here. The package name is only used when loading a binary, which requires the scenario to know which package we are loading up. Here there is no binary, only the unreal editor.

File: example.py (hunk 1 above)
Comment (jaydenmilne) [reply example.py]: Twas

File: example.py (hunk 2 above)
Comment (jaydenmilne) [reply example.py]: Basically, that is only needed when specifying a configuration that isn't a JSON file, correct @daniekpo ?

**Pull request merged**